config/configuration.ts

```diff
@@ -0,0 +1,17 @@
+export const config = {
+  sendGrid: {
+    api_key: process.env.SENDGRID_API_KEY,
+    from: "klaudiusz.witt@gmail.com",
+  },
+  database: {
```

Comment:
If the project requires any external dependencies / libraries mention it in README